common/djangoapps/entitlements/api/v1/views.py

```diff
@@ -51,3 +54,105 @@ def perform_destroy(self, instance):
             )
         if save_model:
             instance.save()
+
+
+class EntitlementEnrollmentViewSet(viewsets.GenericViewSet):
+    authentication_classes = (JwtAuthentication, SessionAuthentication,)
+    permission_classes = (permissions.IsAuthenticated,)
+    queryset = CourseEntitlement.objects.all()
+    serializer_class = CourseEntitlementSerializer
```

Comment:
Since this is an enrollment endpoint, would it make sense to also serialize the enrollment that gets created/updated/destroyed?